### PR TITLE
Edits to README based on edits made to README for ``ansys-api-platfor…

### PR DESCRIPTION
**Pull request by PipKat**
I reviewed a PR (#1!) that Pierre L. made for the ansys-api-platform-instancemanagement repository. He said that he basically used text in the README from this repsository and suggested that I edit its README so that I don't keep having to repeatedly edit the same content when other "lazy" developers reuse it! I've cleaned up this README and run pre-commit locally. However, mypy is failing. I don't know what this is. @akaszynski or @greschd, can either of you help me out?

### CODE REVIEW

**Comment by akaszynski** [comment]
Fixed pre-commit @PipKat.

**Comment by PipKat** [comment]
> Fixed pre-commit @PipKat.

@akaszynski Now I need someone to review? Anyone special I should tag?

File: README.md [hunk 1]
```diff
@@ -1,96 +1,100 @@
...
+You can then use this package in a "higher-level" Python library.
 
 The directory created by this template is fully compatible with GitHub and can
-be uploaded to the [ansys](https://github.com/ansys) organization to
+be uploaded to the [Ansys](https://github.com/ansys) organization to
 automatically push the Python package to PyPI.
 
 
-## Using the Template
+## Installation dependencies
```
Comment (akaszynski) [inline README.md:24]:
```suggestion
## Installation and usage
```

File: README.md [hunk 2]
```diff
@@ -1,96 +1,100 @@
...
-  For example, if you depend on ``ansys-api-file-transfer`` and ``ansys-api-mapdl``:
+  For example, if you depend on the ``ansys-api-file-transfer`` and ``ansys-api-mapdl`` packages, you
+  would pass these dependencies:
+
   ```json
   {"modules": ["ansys-api-tools-file-transfer", "ansys-api-mapdl-core"]}
   ```
 
-## Maintaining your API repository
+## API repository maintenance
```
Comment (akaszynski) [inline README.md:68]: This section refers to your API repository, I think we should keep the section title as-is.

```suggestion
## Maintaining your API repository
```

File: README.md [hunk 3]
```diff
@@ -1,96 +1,100 @@
...
 
-Once your API repository has been generated, you can initialize a `git` repository inside the new directory. Any changes to the `.proto` files can then be made directly in that repository.
+Once your API repository has been generated, you can initialize a ``git`` repository inside the new directory. Any changes to PROTO files can then be made directly in this repository.
```
Comment (akaszynski) [inline README.md:70]: This needs to still refer to the repository created from this tool.

```suggestion
Once your API repository has been generated, you can initialize a ``git`` repository inside the new directory. Any changes to PROTO files can then be made directly in your repository.
```

File: README.md [hunk 4]
```diff
@@ -1,96 +1,100 @@
...
 
-Details for how the `.proto` compilation works and how it can be controlled from `setup.py` and `pyproject.toml` can be found in the `ansys-tools-protoc-helper` [documentation](https://github.com/ansys/ansys-tools-protoc-helper).
+For information on how the compilation of PROTO files works and how compilation can be controlled from the ``setup.py`` and ``pyproject.toml`` files, see the [documentation](https://github.com/ansys/ansys-tools-protoc-helper) for the ``ansys-tools-protoc-helper`` package.
```
Comment (akaszynski) [inline README.md:72]:
```suggestion
For information on how the compilation of PROTO files works and how compilation can be controlled from the ``setup.py`` and ``pyproject.toml`` files, see the [documentation](https://ansys.github.io/ansys-tools-protoc-helper/) for the ``ansys-tools-protoc-helper`` package.
```

**Review by akaszynski — APPROVED** [review]

**Pull request merged**